packages/core/src/external/opensearch/fhir-ingestor.ts

```diff
@@ -202,10 +215,19 @@ export class OpenSearchFhirIngestor {
       log(`Deleting resources from index ${indexName}...`);
       const startedAt = Date.now();
 
-      await client.deleteByQuery({
-        index: indexName,
-        body: createDeleteQuery({ cxId, patientId }),
-      });
+      await executeWithNetworkRetries(
+        async () => {
+          await client.deleteByQuery({
+            index: indexName,
+            body: createDeleteQuery({ cxId, patientId }),
+          });
+        },
+        {
+          httpStatusCodesToRetry: ERROR_CODES_TO_RETRY,
+          initialDelay: INITIAL_WAIT_TIME_BETWEEN_RETRIES.asMilliseconds(),
+          maxAttempts: MAX_RETRY_ATTEMPTS,
```

Comment:
I think we can try more here, prob around 10x or so? Could skip the const at the top and have each place with it's own value (I don't think that'd be a magic number as it's clear by the prop name what it does, but NABD).